lizmap/modules/lizmap/lib/Project/project.php

```diff
@@ -0,0 +1,1567 @@
+     */
+    protected $data = array();
+
+    /**
+     * Version of QGIS which wrote the project.
+     *
+     * @var null|int
+     */
+    protected $qgisProjectVersion;
+
+    /**
+     * @var array contains WMS info
+     */
+    protected $WMSInformation;
+
+    /**
+     * @var string
+     */
+    protected $canvasColor = '';
+
+    /**
+     * @var array authid => proj4
+     */
+    protected $allProj4 = array();
+
+    /**
+     * @var array for each referenced layer, there is an item
+     *            with referencingLayer, referencedField, referencingField keys.
+     *            There is also a 'pivot' key
+     */
+    protected $relations = array();
+
+    /**
+     * @var array list of themes
+     */
+    protected $themes = array();
+
+    /**
+     * @var array list of layer orders: layer name => order
+     */
+    protected $layersOrder = array();
+
+    /**
+     * @var array
+     */
+    protected $printCapabilities = array();
+
+    /**
+     * @var array
+     */
+    protected $locateByLayer = array();
+
+    /**
+     * @var array
+     */
+    protected $formFilterLayers = array();
+
+    /**
+     * @var array
+     */
+    protected $editionLayers = array();
+
+    /**
+     * @var array
+     */
+    protected $attributeLayers = array();
+
+    /**
+     * @var bool
+     */
+    protected $useLayerIDs = false;
+
+    /**
+     * @var array List of cached properties
+     */
+    protected $cachedProperties = array('WMSInformation', 'canvasColor', 'allProj4',
+        'relations', 'themes', 'layersOrder', 'printCapabilities', 'locateByLayer', 'formFilterLayers',
+        'editionLayers', 'attributeLayers', 'useLayerIDs', 'layers', 'data', 'cfg', 'qgisProjectVersion', );
+
+    /**
+     * @var string
+     */
+    private $spatialiteExt;
+
+    /**
+     * version of the format of data stored in the cache.
+     *
+     * This number should be increased each time you change the structure of the
+     * properties of qgisProject (ex: adding some new data properties into the $layers).
+     * So you'll be sure that the cache will be updated when Lizmap code source
+     * is updated on a server
+     */
+    const CACHE_FORMAT_VERSION = 1;
+
+    /**
+     * @var projectCache
+     */
+    protected $cacheHandler;
+
+    /**
+     * constructor.
+     *
+     * @param string           $key      : the project name
+     * @param lizmapRepository $rep      : the repository
+     * @param jelixInfo        $jelix    the instance of jelixInfos
+     * @param mixed            $services
+     */
+    public function __construct($key, $rep, $jelix, $services)
+    {
+        $this->key = $key;
+        $this->repository = $rep;
+        $this->jelix = $jelix;
+        $this->services = $services;
+
+        $file = $this->getQgisPath();
+
+        // Verifying if the files exist
+        if (!file_exists($file)) {
+            throw new UnknownLizmapProjectException('The QGIS project '.$file.' does not exist!');
```

Comment:
careful: PHP will try to find the class `UnknownLizmapProjectException` into the current namespace, `Lizmap\Project`, whereas the class is in the root namespace. So you should indicate `\UnknownLizmapProjectException`.

Fix similar errors in other places